kong/assets/configuration/spec.yaml

```diff
@@ -8,13 +8,13 @@ files:
   - template: instances
     options:
     - name: kong_status_url
-      required: true
```

Comment:
But it is required
https://github.com/DataDog/integrations-core/blob/e668209b4448614c2fd8fadf6467252e4c0d9c1e/kong/datadog_checks/kong/kong.py#L49

Reply:
Oh, but only if openmetrics_endpoint is not defined